<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="SAIMULE" failures="1" tests="1" name="TestRunner.TestRunner" time="14.361" errors="0" timestamp="2025-03-11T10:08:29 IST" skipped="0">
  <testcase classname="TestRunner.TestRunner" name="runScenario" time="14.361">
    <failure message="expected [manda user] but found [Nilesh Kumar]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [manda user] but found [Nilesh Kumar]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at StepDefinition.Sample2Scenario.user_checks_the_functionality_of_login(Sample2Scenario.java:29)
at âœ½.User checks the functionality of login(file:///C:/Users/saimu/eclipse-workspace/ProjectName-3/src/test/resources/Feature/Sample3.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- TestRunner.TestRunner -->
